### PR TITLE
PROGRAMMES-6504 Use Podcast list from DB

### PR DESCRIPTION
**Pull request by votemike**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6504

I've been testing using http://sandbox.bbc.co.uk/programmes/b0b89gc6

### CODE REVIEW

File: src/Controller/FindByPid/ClipController.php [hunk 1]
```diff
@@ -99,16 +99,15 @@ public function __invoke(
             $contributions = $contributionsService->findByContributionToProgramme($clip);
         }
 
-        $rmsPodcastPromise = new FulfilledPromise(null);
-        if ($clip->getTleo() instanceof ProgrammeContainer && $clip->getTleo()->isRadio()) {
-            $rmsPodcastPromise = $podcastService->getPodcast($clip->getTleo()->getPid());
+        $podcast = null;
+        if ($clip->getTleo() instanceof ProgrammeContainer && $clip->getTleo()->isPodcastable()) { //@TODO is that correct?
```
[the remark this answers is not in the document]
Comment (BjornTwachtmann) [reply src/Controller/FindByPid/ClipController.php]: Looks correct to me

**Comment by BjornTwachtmann** [comment]
@votemike This looks good to me. I think we should get rid of the lazy-loaded partial here too really, there's no reason to have it that I can see now all this stuff is in the DB. 
Either merge this and stick a ticket in the next sprint to refactor that away, or add it on to this PR if you prefer.

**Comment by votemike** [comment]
Test this please

**Comment by votemike** [comment]
@BjornTwachtmann when you say "get rid of"... do you mean delete the whole thing? Or keep it, but stop the lazy-loading aspect of it?

**Comment by BjornTwachtmann** [comment]
@votemike , yeah, keep the panel but get rid of the lazy-load

**Pull request merged**